src/search/engines/yep/options.go

```diff
@@ -21,7 +21,6 @@ var dompaths = engines.DOMPaths{
 */
 
 var Support = engines.SupportedSettings{
-	Locale:                  true,
-	SafeSearch:              true,
-	RequestedResultsPerPage: true,
```

Comment:
Removed because it's not *really* working as we want it, because of repeated results and weird logic...
Examples:
- when given a number less than 20 it will work as expected
- when given a number like 31 it will show 20 completely new results
- when given a number like 41 it will show repeated first 20 results and new 20 results that aren't the same as value 31 results